.bibtoolrsc

```diff
@@ -70,7 +71,7 @@ rewrite.rule {"^ *\([0-9]*\) *$" "{\1}"}
 rewrite.rule {"^{ *}$"}
 
 # Unify page range separator
-rewrite.rule {pages "\([0-9]+\) *\(-\|---\|–\) *\([0-9]+\)" "\1--\3"}
+rewrite.rule { pages "\([0-9]+\) *\(-\|---\|–\) *\([0-9]+\)" "\1--\3"}
```

Comment:
If you insert the leading space, then also the trailing space...
```suggestion
rewrite.rule { pages "\([0-9]+\) *\(-\|---\|–\) *\([0-9]+\)" "\1--\3" }
```
... or don't touch that line at all, as it is an unrelated change? Note that the rules *above* this don't have the space, too